components/31-molecules/quote/_quote.scss

```diff
@@ -1,140 +1,84 @@
-.quote-wrapper {
-  overflow: hidden;
-}
-
 .quote {
   @include theme('border-color', 'color-primary--lighten-4', 'quote-border-color');
+  margin-top: 3.5rem;
```

Comment:
A long time ago, we decided to make all outer margins the responsibility of the layout, not of the component.
That's probably why there was a quote-wrapper.

Reply:
I hear you 👍